Source/Core/ViewModel/SignInViewModel.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+public class SignInViewModel {
+    
+    // MARK: Public Instance Properties
+    
+    public let domain: String
+    
+    // MARK: Public Initializers
+    
+    public init() {
+        
+        self.domain = "andela.com"
```

Comment:
I think the domain can go to the localized file and be referenced from there

Reply:
Does this file exist yet?

Reply:
Never mind found it

Reply:
What is the purpose of this anyway? It's not like andela.com will be a different domain name in say French or German